assets/wizards/popups/views/segmentation/SingleSegment.js

```diff
@@ -247,7 +295,7 @@ const SingleSegment = ( { segmentId, setSegments, wizardApiFetch } ) => {
 			) }
 
 			<div className="newspack-buttons-card">
-				<Button disabled={ ! isSegmentValid } isPrimary onClick={ saveSegment }>
+				<Button disabled={ ! isSegmentValid || ! isDirty } isPrimary onClick={ saveSegment }>
```

Comment:
Can we add a condition to disable the button if the segment config matches the default config? This might discourage people from accidentally creating segments that are the same as "no segment".

Reply:
Good idea, done in 3c5238bdd6e56efa27df563b251afad8223d2a80